app/javascript/map.js

```diff
@@ -85,6 +89,40 @@ class Map {
     }
   }
 
+  placeLocationsInBounds() {
+    setTimeout(() => {
```

Comment:
We definitely don't want a timeout here.  I am confused why we should have a timeout.